src/style.css

```diff
@@ -5,21 +5,30 @@
 
 body {
   font-family: Arial, Helvetica, sans-serif;
+  height: 90vh;
+  background-color: #fff0d1;
+}
+
+h2 {
+  color: #5f5a4e;
```

Comment:
[OPTIONAL] I think you forgot to add MIT license to your project as well. Kindly add them as well.